lib/validator/v3/validator.go

```diff
@@ -742,38 +749,90 @@ func validateProtoPort(v *validator.Validate, structLevel *validator.StructLevel
 			reflect.ValueOf(m.Protocol),
 			"ProtoPort.Protocol",
 			"",
-			reason("ProtoPort protocol must be 'TCP' or 'UDP'."),
+			reason("protocol must be 'TCP' or 'UDP'."),
 		)
 	}
 }
 
-func validateObjectMeta(current interface{}, oma v1.ObjectMetaAccessor) error {
-	if _, ok := current.(*api.GlobalNetworkPolicy); ok {
-		// Must begin and end with a letter or number, dots are disallowed,
-		// underscore and dash are allowed.
-		matched := nameNoDotsRegex.MatchString(oma.GetObjectMeta().GetName())
-		if !matched {
-			return cerrors.ErrorValidation{
-				ErroredFields: []cerrors.ErroredField{{
-					Name:   "ObjectMeta.Name",
-					Reason: "Name must begin and end with a letter or number, dots are not allowed, dashes and underscores are allowed.",
-					Value:   oma.GetObjectMeta().GetName(),
-				}},
-			}
+func validateObjectMeta(v *validator.Validate, structLevel *validator.StructLevel) {
+	om := structLevel.CurrentStruct.Interface().(metav1.ObjectMeta)
+
+	// Must begin and end with a letter or number, dot, underscore,
```

Comment:
This comment looks wrong ... 
`Must begin and end with lower case letter or number; dots, underscores, and dashes are allowed`